engine/apps/phone_notifications/zvonok_phone_provider.py

```diff
@@ -0,0 +1,67 @@
+import requests
+import os
+
+from random import randint
+
+from django.core.cache import cache
+
+from apps.base.models import LiveSetting
+from apps.base.utils import live_settings
+
+from .phone_provider import PhoneProvider, ProviderFlags
+
+ZVONOK_CALL_URL = 'https://zvonok.com/manager/cabapi_external/api/v1/phones/call/'
+
+class ZvonokPhoneProvider(PhoneProvider):
+    """
+    ZvonokPhoneProvider is an implementation of phone provider which supports only voice calls (zvonok.com).
+    API docs: https://api-docs.zvonok.com/ . Call status description: https://zvonok.com/ru-ru/guide/guide_statuses/
+    """
+
+    def make_notification_call(self, number, message):
+        url = ZVONOK_CALL_URL
+        payload={'public_key': ZVONOK_API_KEY,
+        'phone': number,
+        'campaign_id': ZVONOK_CALL_CAMPAIGN_ID,
+        'text': message,
+        'speaker': 'Joanna'}
+        response = requests.request("POST", url, data=payload)
+        call_id = response.json().get("call_id")
+        print(f'call_id: {call_id}')
```

Comment:
Could you please use logger, instead of print? The logline will look like:
`logger.info(f"ZvonokPhoneProvider.make_notification_call: .......")`

Reply:
Done